custom/icds_reports/reports/awc_infrastracture.py

```diff
@@ -157,8 +160,52 @@ def get_data_for(month, filters):
                     'format': 'percent_and_div',
                     'frequency': 'month',
                     'redirect': 'awc_infrastructure/medicine_kit'
-                }
+                },
+                {
+                    'label': _('AWCs Reported Infantometer'),
+                    'help_text': awcs_reported_infantometer_text(),
+                    'percent': percent_diff(
+                        'infantometer',
+                        this_month_data,
+                        prev_month_data,
+                        'sum_last_update'
+                    ),
+                    'color': get_color_with_green_positive(percent_diff(
+                        'infantometer',
+                        this_month_data,
+                        prev_month_data,
+                        'sum_last_update'
+                    )),
+                    'value': get_value(this_month_data, 'infantometer'),
+                    'all': get_value(this_month_data, 'sum_last_update'),
+                    'format': 'percent_and_div',
+                    'frequency': 'month',
+                    'redirect': 'awc_infrastructure/infantometer'
+                },
             ],
```

Comment:
This error is single row of two tiles? @challabeehyv   and thats why these all are segregated into array of two elements?

Reply:
@Rohit25negi I didn't get you. Can you please elaborate?

Reply:
@challabeehyv  
like  here the 2 jsons  are combined as an array.   Like "AWCs Reported Weighing Scale: Infants" and "AWCs Reported Weighing Scale: Mother and Child" in one array, then "AWCs Reported Medicine Kit" and "AWCs Reported Infantometer" in one array.
Is this because each array represent set of two tiles on UI fitted along side of each other? and why stadiometer is added is another array?